tools/lib/provision.py

```diff
@@ -425,10 +426,10 @@ def main(options: argparse.Namespace) -> "NoReturn":
                         default=False,
                         help="Ignore all provisioning optimizations.")
 
-    parser.add_argument('--production-travis', action='store_true',
-                        dest='is_production_travis',
+    parser.add_argument('--production-circle-ci', action='store_true',
+                        dest='is_production_circle_ci',
                         default=False,
-                        help="Provision for Travis with production settings.")
+                        help="Provision for Circle CI with production settings.")
```

Comment:
I think this would probably be better named `--production-test-suite`; we don't need to emphasize the name of the testing platform in this option name, as it's not the important detail here.